src/Common/tests/System/RealFormatterTestsBase.netcoreapp.cs

```diff
@@ -14,40 +14,37 @@ public abstract class RealFormatterTestsBase
         protected abstract string InvariantToStringDouble(double d, string format);
         protected abstract string InvariantToStringSingle(float f, string format);
 
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/35566")]
-        [Theory]
-        [InlineData(double.Epsilon, "�0.00")]
-        [InlineData(double.MaxValue, "�179,769,313,486,231,570,814,527,423,731,704,356,798,070,567,525,844,996,598,917,476,803,157,260,780,028,538,760,589,558,632,766,878,171,540,458,953,514,382,464,234,321,326,889,464,182,768,467,546,703,537,516,986,049,910,576,551,282,076,245,490,090,389,328,944,075,868,508,455,133,942,304,583,236,903,222,948,165,808,559,332,123,348,274,797,826,204,144,723,168,738,177,180,919,299,881,250,404,026,184,124,858,368.00")]
-        [InlineData(Math.E, "�2.72")]
-        [InlineData(Math.PI, "�3.14")]
-        [InlineData(0.0, "�0.00")]
-        [InlineData(0.84551240822557006, "�0.85")]
-        [InlineData(1.0, "�1.00")]
-        [InlineData(1844674407370955.25, "�1,844,674,407,370,955.25")]
+        [Theory]
+        [InlineData(double.Epsilon, "¤0.00")]
```

Comment:
All these lines are marked as "changed" since the encoding was changed from ASCII to UTF-8.